specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-02-09/Quotas.json

```diff
@@ -0,0 +1,313 @@
+                        "$ref": "examples/Quotas/Create.json"
+                    }
+                },
+                "tags": [
+                    "Quotas"
+                ],
+                "description": "Creates or Updates a Quota.",
+                "summary": "Creates or Updates a Quota.",
+                "operationId": "Quotas_CreateOrUpdate",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/QuotaParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/NewQuotaParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "Quotas.json#/definitions/Quota"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "x-ms-examples": {
+                    "Deletes specified quota.": {
+                        "$ref": "examples/Quotas/Delete.json"
+                    }
+                },
+                "tags": [
+                    "Quotas"
+                ],
+                "summary": "Deletes specified quota",
+                "description": "Delete an existing quota.",
+                "operationId": "Quotas_Delete",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/QuotaParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK"
+                    },
+                    "404": {
```

Comment:
404 or 204(NoContent) for delete operations ? Typically for all the tracked resources Delete on non existing should return 204

Reply:
Remove the 404.

Reply:
Remove 404

Reply:
Yes, Error codes shouldn't be there in the swagger specs. Im curious what the error code should be for the Delete Operation here.. 404 or 204?

Reply:
Doesn't really matter, just accept a 200/202 if you return them as those are non-error codes.